doc/overview.rst

```diff
@@ -1,98 +1,106 @@
+########
 Overview
-========
+########
 
 .. image:: images/mainwindow/mainwindow_custom_1610.png
    :align: center
    :alt: The Spyder main window, with numerous panels open in a custom layout
 
 |
 
-Spyder is a Python development environment with the following key features:
+Spyder, the Scientific Python Development Environment, includes the following key features:
 
-Key features:
+* :doc:`editor`:
+
+  * Customizable syntax highlighting themes (Python, C/C++, Fortran)
+  * :doc:`debugging` breakpoints and conditional breakpoints (through ``ipdb`` integration)
+  * Interactive execution: Run line/selection, run cell, run file, re-run and debug
+  * Run configuration settings:
+
+    * Working directory selection
+    * Command line options
+    * Run in Current/dedicated/external console choice
+    * Automatically clear variables or enter debugging
+
+  * :guilabel:`Outline Explorer`: Navigate cells, functions, classes, blocks, and more
+  * Real-time code introspection features (powered by ``rope`` and ``jedi``):
 
-* general features:
+    * On-demand (:kbd:`Tab`) and "dot" automatic code completion
+    * Automatic popup calltips showing function signatures
+    * Go-to-definition for any symbol: Functions, classes, attributes, etc. (:kbd:`Ctrl`-Click or :kbd:`Ctrl-G` by default)
 
-  * MATLAB-like PYTHONPATH management dialog box (works with all consoles)
-  * Windows only: current user environment variables editor
-  * direct links to documentation (Python, Matplotlib, !NumPy, !Scipy, etc.)
-  * direct link to Python(x,y) launcher
-  * direct links to !QtDesigner, !QtLinguist and !QtAssistant (Qt documentation)
+  * Occurrence highlighting: Select or double-click any word to show all other instances throughout the current document
+  * On-the-fly automatic formatters (optional):
 
-* *preferences* dialog box:
+    * Automatic insertion of closing quotes parentheses, braces and brackets
+    * Automatic indentation after 'else', 'elif', 'finally', etc.
+    * Smart auto-indentation based on code structure
+    * Automatic insertion of colons after for, if, def, etc.
+    * Automatically fix mixed indentation, EoL characters and trailing spaces
```

Comment:
I used EoL since ``of`` is not capitalized in Title Case (and thus generally shouldn't be in the acronym, to reflect its origins), but it seems the convention is ``EOL`` (``EoL`` being more associated with End of Life). Fixed.